vue-frontend/src/components/home-new/CTASection.vue

```diff
@@ -1,78 +1,116 @@
+        <div class="v2-header-3-text canopas-gradient-text">
+          How do you know we’re a good fit for you?
+        </div>
+        <div class="v2-title-3-text mt-4">Let’s have a short chat.</div>
+        <div class="communication">
+          <div class="item px-4 py-3">
+            <div class="v2-normal-3-text">I love talking.</div>
+            <router-link class="gradient-btn cta-button" :to="contactURL">
+              <font-awesome-icon
+                class="fa v2-title-3-text"
+                icon="phone"
+                aria-hidden="true"
+              />
+              <span class="v2-normal-3-text">Schedule a call</span>
+            </router-link>
+          </div>
+          <div class="item px-4 py-3">
+            <div class="v2-normal-3-text">I prefer mail.</div>
+            <router-link class="gradient-btn cta-button" :to="contactURL">
+              <font-awesome-icon
+                class="fa v2-title-3-text"
+                icon="paper-plane"
+                aria-hidden="true"
+              />
+              <span class="v2-normal-3-text">Drop a mail</span>
+            </router-link>
+          </div>
         </div>
-        <div class="item px-4 py-3">
-          <div class="v2-normal-3-text">I prefer mail.</div>
-          <router-link class="gradient-btn cta-button" :to="contactURL">
-            <font-awesome-icon
-              class="fa v2-title-3-text"
-              icon="paper-plane"
-              aria-hidden="true"
-            />
-            <span class="v2-normal-3-text">Drop a mail</span>
-          </router-link>
+        <div
+          class="v2-title-3-text text-left v2-canopas-gradient-text reason-text mt-5"
+        >
+          Four reasons why do we do this call?
         </div>
+        <ul class="v2-normal-2-text chat-list text-left mt-4">
+          <li>
+            <span class="bullet"></span>
+            <div>It gives an opportunity to learn more about each other.</div>
+          </li>
+          <li>
+            <span class="bullet"></span>
+            <div>
+              We want to give you a chance to put your idea into action. You
+              might have faced this situation before where you wish you had
+              someone to help you.
+            </div>
+          </li>
+          <li>
+            <span class="bullet"></span>
+            <div>
+              Sometimes, you want to check the feasibility of your idea. We will
+              give you honest feedback based on our industry experience.
+            </div>
+          </li>
+          <li>
+            <span class="bullet"></span>
+            <div>
+              We offer 100% Money-back guarantee. so, there’s no risk to you.
+            </div>
+          </li>
+        </ul>
       </div>
       <div
-        class="v2-title-3-text text-left v2-canopas-gradient-text reason-text mt-5"
+        v-if="currentRoutePath == '/portfolio/' + id ? 'showCopyrightDiv' : ''"
```

Comment:
This v-if condition is needed ??

Reply:
yes, in order to hide copyright div when we are on home page and show when redirect to portfolio page

Reply:
we can also manage that condition in below class binding css

Reply:
Yesterday i have done that using visibility: hide class in binding but you told to hide that whole div that's why I have used v-if. Should I change it again?